manifests/testing/file-host.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: cdi-file-host
```

Comment:
we likely want this deployed in the same Namespace as the other cdi components as well.